styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -95,6 +95,42 @@
   display: none;
 }
 
+.refreshBanner-homepage-062018 {
+  background-image: url('../imgs/rs-homepage-background-06_2018.jpg');
+  height: 680px;
+}
+
+.refreshBannerCombined-newheader-062018 {
+  margin-left: 50px;
+  margin-top: 180px;
+  width: 425px;
+}
+
+.refreshBannerCombined-ctaHeadline-062018 {
+  display: block;
+  font-size: 6.25rem;
+}
+
+.rsHomepage-subHeader-062018 {
+  color: $white;
+  font-size: 3.25rem;
+  line-height: 3.25rem;
+}
+
+.rsHomepage-button-062018 {
+  &.button {
+    &.ghost {
```

Comment:
So this gets output as `.rsHomepage-button-062018.button.ghost`, which I think is one selector more than necessary here.  I think `.rsHomepage-button-062018.ghost` is sufficient as it occurs higher in the document than `.button.ghost` which we're trying to override.

Reply:
Sorry, I meant lower, now higher.